### ISSUE TITLE
Learn jetpack compose

### ISSUE THREAD

**Issue by YigitSekerci** [bounswe/bounswe2023group4#181]
### Problem Definition

Jetpack compose will be widely used in our application's UI so we must learn it beforehand. Every member should learn basics of the compose from online sources. Since it is very similar the to the other functional UI coding frameworks like flutter and react, those who have experienced one of the languages can familiarize themselves with compose in no time. If you know these other languages you may look comparison videos for easy migration.

### Problem Context

* This issue is one of the divided issues for mobile research which are indicated in https://github.com/bounswe/bounswe2023group4/issues/176.

### Acceptance Criteria

* Learn basics of jetpack compose.

### Suggested Solutions

* https://www.youtube.com/@PhilippLackner This YouTuber has good videos about jetpack compose you may look him.

**Comment by YigitSekerci** [comment]
I have already known the jetpack compose but I have looked anyway for the experimental api changes in latest compose version for easy adaptation to changes.

**Comment by aemresafak** [comment]
I have learned what jetpack compose is and how it is different than views. I learned what declarative UI is what state is what row and column is etc. I am good to go.

**Comment by AliAlperenSonmez** [comment]
I have successfully acquired the basics of Jetpack Compose, equipping me with the necessary knowledge to contribute effectively to our application's UI development. I have explored valuable resources, such as the videos by Philipp Lackner on YouTube, to enhance my understanding of Jetpack Compose, ensuring I am well-prepared to use this technology in our project.
For those with prior experience in functional UI coding frameworks like Flutter and React, I have leveraged the similarities to quickly adapt to Jetpack Compose, making the learning process efficient and seamless.
